crates/pop-cli/tests/parachain.rs

```diff
@@ -121,26 +140,28 @@ default_command = "./target/release/parachain-template-node"
 
 [[parachains.collators]]
 name = "collator-01"
-"#,
-			random_port
+rpc_port = {random_port}
+"#
 		),
 	)?;
 
 	// `pop up network -f ./network.toml --skip-confirm`
-	let mut cmd = Cmd::new(cargo_bin("pop"))
-		.current_dir(&temp_parachain_dir)
-		.args(&["up", "network", "-f", "./network.toml", "--skip-confirm"])
-		.spawn()
-		.unwrap();
+	let mut command = pop(
+		&working_dir,
+		&["up", "network", "./network.toml", "-r", "stable2412", "--verbose", "--skip-confirm"],
```

Comment:
2503-1 doesnt produce blocks with the available paseo-local runtimes, resulting in the call steps failing.